release_notes/RELEASE.txt

```diff
@@ -47,7 +47,10 @@ New features and changes
       Also added a section to INSTALL_CMake.txt with very basic explanation of the
       process to use CMakePresets.
 
+    - Private headers are no longer deployed
 
+      * dfi.h - Declared private, unused, undocumented in UG/RM/DG, has been
+                removed from the library entirely.
```

Comment:
I'll expand this list in future commits as we move private headers out of the deploy list